spec/features/run_search_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+feature 'User searches for a weather', %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.